internal/provider/resource_tfe_organization.go

```diff
@@ -193,8 +200,15 @@ func resourceTFEOrganizationUpdate(d *schema.ResourceData, meta interface{}) err
 	}
 
 	// If send_passing_statuses_for_untriggered_speculative_plans is supplied, set it using the options struct.
-	if sendPassingStatusesForUntriggeredSpeculativePlans, ok := d.GetOk("send_passing_statuses_for_untriggered_speculative_plans"); ok {
-		options.SendPassingStatusesForUntriggeredSpeculativePlans = tfe.Bool(sendPassingStatusesForUntriggeredSpeculativePlans.(bool))
+	if d.HasChange("send_passing_statuses_for_untriggered_speculative_plans") {
+		_, newVal := d.GetChange("send_passing_statuses_for_untriggered_speculative_plans")
+		options.SendPassingStatusesForUntriggeredSpeculativePlans = tfe.Bool(newVal.(bool))
+	}
+
+	// If aggregated_commit_status_enabled is supplied, set it using the options struct.
+	if d.HasChange("aggregated_commit_status_enabled") {
+		_, newVal := d.GetChange("aggregated_commit_status_enabled")
+		options.AggregatedCommitStatusEnabled = tfe.Bool(newVal.(bool))
 	}
```

Comment:
Due to issues with`GetOk` and resolving zero value for booleans, workaround to only include attribute if it has been altered. Prevents issues with `aggregated_commit_status_enabled = true` changed to `aggregated_commit_status_enabled = false`. Whereas it was previously omitted.

Reply:
This workaround makes sense to me! I think it's more readable than using `GetOk` with an `else` block, which confused me a bit at first.